triage.md

```diff
@@ -0,0 +1,68 @@
+## Table of Contents
+- [Scope](#scope)
+- [Schedule](#schedule)
+- [What Is Triaging](#what-is-triaging)
+- [How to Triage](#how-to-triage)
+
+## Scope
+These guidelines serve as a primary document for triaging issues in [opensearch-benchmark](https://github.com/opensearch-project/opensearch-benchmark) and [opensearch-benchmark-workloads](https://github.com/opensearch-project/opensearch-benchmark-workloads) project. For maintainers/contributors, it is a guide to ensure that we address all customer issues and address feature requests in a timely manner.
```

Comment:
This document provides guidelines for the triaging of GitHub issues in the ... and ... repositories.